server/views/components/primary-navigation/macro.njk

```diff
@@ -0,0 +1,15 @@
+{%- from "moj/components/primary-navigation/macro.njk" import mojPrimaryNavigation -%}
+
+{%- macro primaryNavigation(params) %}
+    {% set regionNameHTML %}
+      <div class="cas-primary-navigation__regional_header">
+        {{ params.regionName | escape }}
+      </div>
+    {% endset %}
+
+    {{ mojPrimaryNavigation({
+        label: 'Primary navigation',
+        items: [],
+        searchHtml: regionNameHTML
+    }) | safe | trim | indent(8)  }}
```

Comment:
Not sure these template filters applied to the generated markup are useful? This only seems to make the template itself harder to parse (I had to stop and think about what they were for), and I can't think of a scenario where one may access the source without a parser of sorts (dev tools, etc).